src/components/MovieInfo.jsx

```diff
@@ -0,0 +1,19 @@
+export const MovieInfo = ({ id, poster, title, vote, description }) => {
+    // This component generates a section for displaying details of a movie. It includes an image, the movie's title, a user rating, and a description.
```

Comment:
Great! Maybe you could have added a comment on where this component is added as a comment?